content/html/concepts/attributes/style/style.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Can we create a `terms/` folder within `attributes/` and move the `style/style.md` folder and file under that terms folder? This just happens to be the structure of Docs. If you need assistance with this, please let me know and I can assist.

Reply:
Done